src/hooks/screen/useChatScreenHook.ts

```diff
@@ -44,6 +45,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
 
   const sendChat = async (
     message: string = randomString(20),
+    attachments: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` parameter is initialized as an empty array. If this function is expected to handle cases where no attachments are provided, it would be better to default to `null` or `undefined` and handle these cases accordingly in the function body.
```diff
-     attachments: [],
+     attachments: null,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->